.github/workflows/js_ci.yml

```diff
@@ -1,32 +1,14 @@
 name: JS
+
 on:
+  push:
+    branches:
+      - master
   pull_request:
-    paths:
-      - 'webpack/**'
-      - 'package.json'
-      - '.github/workflows/js_ci.yml'
+
 jobs:
   test_js:
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        node-version: [14]
-    steps:
-      - uses: actions/checkout@v2
-      - name: Setup Node
-        uses: actions/setup-node@v1
-        with:
-          node-version:  ${{ matrix.node-version }}
-      - name: Npm install
-        run: |
-          npm install
-      - name: Run plugin linter
-        run: |
-          npm run lint
-      - name: Run custom plugin linter
-        run: |
-          npm run lint:custom
```

Comment:
this is currently not executed by the shared action, but I thought `tfm-lint` would already run eslint?

Reply:
@MariaAga ^ opinions?

Reply:
I was gonna do this pr anyway after I make changes in the shared actions, which will run the lint custom.
I'm making changes in the shared actions after: https://github.com/theforeman/foreman/pull/10239